### PR TITLE
fix: remove i18n from title and description

### LINKED ISSUE #4296: remove i18n from title and description in Meta
we need to look at each Meta component and remove the `t("key_string")` and hard-code the english version

https://user-images.githubusercontent.com/8019099/189156327-89a7c6c2-246a-4d9f-90db-23e042ed9d1d.mp4

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: #4296

### CODE REVIEW

**Review by hariombalhara — APPROVED** [review]

**Comment by hariombalhara** [comment]
@Udit-takkar Can you please resolve the conflicts

File: apps/web/pages/v2/settings/admin/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { getLayout } from "@calcom/ui/v2/core/layouts/AdminLayout";
 function AdminAppsView() {
   return (
     <>
-      <Meta title="admin" description="admin_description" />
+      <Meta title="Admin" description="admin_description" />
```
Comment (PeerRich) [inline apps/web/pages/v2/settings/admin/index.tsx:7]: still missing here

File: apps/web/pages/v2/settings/admin/users.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { getLayout } from "@calcom/ui/v2/core/layouts/AdminLayout";
 function AdminUsersView() {
   return (
     <>
-      <Meta title="users" description="users_description" />
+      <Meta title="Users" description="users_description" />
```
Comment (PeerRich) [inline apps/web/pages/v2/settings/admin/users.tsx:7]: still missing here

**Review by PeerRich — DISMISSED** [review]
three descriptions missing

**Comment by Udit-takkar** [comment]
@PeerRich  these three does not exist in /en/common.json

**Comment by Udit-takkar** [comment]
I have resolved conflicts. just these three key are missing from en/common.json

cc: @PeerRich

**Pull request merged**